### PR TITLE
Add support for Linux ARM64

### PR DESCRIPTION
**Pull request by martin-g**
Then we will use `@conda-forge-admin, please rerender` to re-create the CI config files

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by martin-g** [comment]
@conda-forge-admin, please rerender

**Comment by martin-g** [comment]
@conda-forge-admin, please rerender

**Comment by martin-g** [comment]
@h-vetinari How can I add the linux-aarch64 CI files for blis ?
I thought that since `blis` is listed at https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/arch_rebuild.txt#L43 then `@conda-forge-admin, please rerender` will know that it should add the linux-aarch64 CI support files, but it does not work this way.
I tried to add the file manually with https://github.com/conda-forge/blis-feedstock/pull/29/commits/5c70b4757b2a86a22db1f1ed263531851314f0a3 but the rerendering removes it.

**Comment by h-vetinari** [comment]
> How can I add the linux-aarch64 CI files for blis ?

Easiest would have been to look what #27 did ;)

In any case, I added the necessary config now.

**Comment by h-vetinari** [comment]
> I thought that since `blis` is listed at https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/arch_rebuild.txt#L43 then [...]

The fact that the bot hasn't opened a PR itself means that almost certainly there are missing dependencies (that haven't yet been built for aarch). You'll see this in the CI here anyway if so.

File: conda-forge.yml [hunk 1]
```diff
@@ -1,5 +1,8 @@
+build_platform:
```
Comment (h-vetinari) [inline conda-forge.yml:1]: FWIW, this specifies that we're cross-compiling aarch from x64. It's possible that it's easier to start from a "native" build (still actually running on x64, but emulated through qemu - this is much slower though)

**Comment by martin-g** [comment]
Thanks!

I will add the changes to `build.sh` tomorrow! 
Currently the Linux aarch64 build due to some SSE related CFLAGS.

I don't see `blis` at https://conda-forge.org/status/migration/aarch64andppc64leaddition <- here I check for missing dependencies.
(I still learn how the things work! :-) )

**Comment by martin-g** [comment]
@h-vetinari Please review !

**Comment by isuruf** [comment]
Please use the actual commits from #27 instead of copying the code.

**Comment by martin-g** [comment]
There are 19 commits at https://github.com/conda-forge/blis-feedstock/pull/27/commits
Many of them are mix between `build.sh` and the (now) auto-generated files by the bot.
It won't be easy to extract only the useful part from them.
Another option is to clone from https://github.com/gdonval/blis-feedstock/tree/aarch64_support and then re-render and do the requested changes.
But I see no point in the exercise :-/

**Comment by martin-g** [comment]
Done with #30

**Pull request closed without merging**